flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobUtils.java

```diff
@@ -161,7 +161,7 @@ static Reference<File> createBlobStorageDirectory(
             if (fallbackStorageDirectory != null) {
                 baseDir = fallbackStorageDirectory.deref();
 
-                if (baseDir.mkdirs() || baseDir.exists()) {
+                if (baseDir.exists() || baseDir.mkdirs()) {
```

Comment:
the first thing that mkdirs() does is call exists().  Can we remove the baseDir.exists()?

Reply:
Thanks for your reminder. But it let me take a second thinking.
After the second thinking, it's no need to change this line.